index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <title>My portoflio</title>
+</head>
+<body>
+    <nav class="nav_bar">
+        <div class="logo"> <h2>Mylogo</h2></div> 
+            </div class="nav_list">
+                <ul>
+                    <li><a href="#">portflio</a></li>
+                    <li><a href="#">about</a></li>
+                    <li><a href="#">contact</a></li>
+                </ul>
+                <img class="burger-icon" src="img/Union.png" alt="bruger-icon">
+
```

Comment:
- [ ] Good work including your burger menu here ✔️. However, please kindly make sure that you wrap your mobile menu using a `nav` semantic tag instead of a `div` tag to make your site more accessible 🤓.